functions/go/starlark/README.md

```diff
@@ -4,15 +4,20 @@
 
 <!--mdtogo:Short-->
 
-Run a Starlark script to update or validate resources.
+Run a Starlark script to mutate or validate resources.
 
 <!--mdtogo-->
 
-### Synopsis
+### FunctionConfig
 
 <!--mdtogo:Long-->
 
-You can specify your Starlark script inline under field source like this:
+[Starlark] is a dialect of Python. It is commonly used as a configuration language. It is an untyped dynamic language with
```

Comment:
This is part of the overview, not Function Config.

In general, this usage doc is very spartan and needs to be expanded.